granary/meetup.py

```diff
@@ -99,3 +100,36 @@ def _create(self, obj, preview=False, include_link=source.OMIT_LINK, ignore_form
 
     def return_error(self, msg):
         return source.creation_result(abort=True, error_plain=msg, error_html=msg)
+
+    def user_to_actor(self, user):
+      """Converts a user to an actor.
+
+      Args:
+        user: dict, a decoded JSON Meetup user
+
+      Returns:
+        an ActivityStreams actor dict, ready to be JSON-encoded
+      """
+      user_id = user.get('id')
+      user_id_str = str(user_id),
+      published_s = round(user.get('joined') / 1000)
+      published_dt = datetime.datetime.fromtimestamp(published_s)
+      return util.trim_nulls({
+        'objectType': 'person',
+        'displayName': user.get('name'),
+        'image': {'url': user.get('photo', {}).get('photo_link')},
+        'id': self.tag_uri(user_id_str),
+        # numeric_id is our own custom field that always has the source's numeric
+        # user id, if available.
+        'numeric_id': user_id,
+        'published': published_dt.isoformat(),
+        'url': self.user_url(user_id_str),
+        'urls': None,
```

Comment:
feel free to drop these `None` fields entirely if you want.

Reply:
I don't mind leaving them so it's clear they're not used? Or are there more fields that are left empty, so it makes more sense to leave them all out?

Reply:
I've removed them